content/docs/user-guide/configuration.md

```diff
@@ -10,22 +10,24 @@ config.
 
 ## Options
 
-- `log_level` - logging level to use. Default `INFO`
-- `debug` - whether to run MLEM in debug mode. Sets `log_level` to `DEBUG`.
-  Default `False`
-- `no_analytics` - whether to stop collecting usage telemetry. Default `False`
-- `default_storage` - where to store saved artifacts by default. Should be
-  serialized `Storage` instance. Default is `LocalStorage`, which means save
-  artifacts locally.
-- `default_external` - whether to save objects as
-  [external](/docs/user-guide/project-structure#External) by default. Default is
-  `False`
-- `emojis` - whether to show 🖇️🦉🤩🇪🇲🅾️🇯ℹ️🇸🤩🦉🖇️ in CLI output. Default ✅
 - `additional_extensions` - comma-separated list of extension modules to
-  force-load on MLEM import.
-- `autoload_exts` - turn on
-  [dynamic extension loading](/doc/object-reference/extending#extension-dynamic-loading).
-  Default `True`
+  force-load on MLEM import. None by default
+- `autoload_exts` - turn on [dynamic extension loading]. Default: `True`
+- `debug` - whether to run MLEM in debug mode; Sets `log_level` to `DEBUG`.
+  Default: `False`
+- `default_external` - whether to save objects as [external] by default.
+  Default value: `False`
+- `default_storage` - where to store saved artifacts by default. Should be a
+  serialized `Storage` instance. The default is `LocalStorage`, which means
+  "save artifacts locally".
+- `emojis` - whether to show 🖇️🦉🤩🇪🇲🅾️🇯ℹ️🇸🤩🦉🖇️ in CLI output. Default: ✅
+- `log_level` - logging level to use. Default: `INFO`
+- `no_analytics` - whether to stop collecting usage telemetry. Default: `False`
+- `state` - allows you to set the `url` for a [remote state manager].
```

Comment:
Are there any other subfields (besides `url`) ?

Reply:
```suggestion
- `state` - allows you to set the `uri` for a [remote state manager].
```
https://github.com/iterative/example-gto/blob/mlem/.mlem.yaml as an example

Reply:
Yes, there are other fields except `uri`, and they can be different depending on the implementation. If `uri` is on local fs, [there are two params](https://github.com/iterative/mlem/blob/66ad87114a9c3a46f372d55b3d9e0dc03f11bcb5/mlem/core/objects.py#L1081) `locking` and `lock_timeout`

For remote ones [there are additionally](https://github.com/iterative/mlem/blob/66ad87114a9c3a46f372d55b3d9e0dc03f11bcb5/mlem/core/objects.py#L1128) `storage_options`.

These may change over time, so alternative would be to mention that specific params that could be set can be found with `mlem types state local` for local storing of the state and `mlem types state fsspec` for remote storing:
```shell
$ mlem types state local
Type mlem.core.objects.LocalFileStateManager
MlemABC parent type: state
MlemABC type: local
StateManager that stores state as yaml file locally
Fields:
[not required] locking: bool = True
        Enable state locking
[not required] lock_timeout: float = 600
        Lock timeout
$ mlem types state fsspec
Type mlem.core.objects.FSSpecStateManager
MlemABC parent type: state
MlemABC type: fsspec
StateManager that stores state as yaml file in fsspec-supported filesystem
Fields:
[required] uri: str
        URI of directory to store state files
[not required] storage_options: Dict[str, any] = {}
        Additional options
[not required] storage_options.key: any = None
        Element of storage_options
[not required] locking: bool = True
        Enable state locking
[not required] lock_timeout: float = 600
        Lock timeout
```

Reply:
We should document all these fields somewhere. Dn't they get printed by the semi-automated cmd ref? Let's merge this for now? (Applied your fix.)

Reply:
They are printed indeed - by `mlem types state fsspec` for example. But here we may just add a hint suggesting to run that cmd to see the output. I'm approving - feel free to either merge or add that link.

Reply:
Ah, I see - you have a link to https://mlem.ai/doc/user-guide/deploying#setting-up-remote-state-manager
so maybe we need to add the hint about `mlem types ...` there?

Reply:
Yeah I just think it's misleading to say that `state` sets a remote state manager `uri` when it can set many other values. Should clarify that AND hink about `mlem types` prob.